src/components/header/Header.vue

```diff
@@ -81,12 +90,31 @@ export default defineComponent({
       setCloseModal,
       setWalletModal,
       openSelectModal,
-      disconnectAccount,
+      changeAccount,
     } = useConnectWallet();
 
     const store = useStore();
     const currentNetworkIdx = computed(() => store.getters['general/networkIdx']);
-    const headerName = computed(() => store.getters['general/headerName']);
+    const route = useRoute();
+    const path = computed(() => route.path.split('/')[1]);
+    const headerName = ref('');
+    watch(
+      path,
+      () => {
+        if (path.value === 'dashboard') {
```

Comment:
maybe to have a lookup table instead a lot of if else statements. It will be easier to maintain later.

Reply:
Design is a dropdown so it should start from where the current wallet name is. But I guess this can be fixed later. It matters more to see a big scroll everywhere when a mouse is connected.

Reply:
great catch. resolved as separate function